api/wasm/cpp/contrib/BUILD

```diff
@@ -0,0 +1,17 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+envoy_package()
+
+envoy_cc_library(
+    name = "contrib_lib",
```

Comment:
Nit: `expr_lib`, in case we have other features in contrib in the future?

Reply:
I think we can split this directory once it becomes large. I like rule name matching the directory name.

Reply:
Sounds good.